src/utils/breach-logo.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param {unknkown} breach
+ * @param {Map<string, string>} logos Map of URLs to logos indexed by the domain name of the respective company
+ * @returns {string} HTML for a breach logo (either an `img`, or a `span.breach-logo` containing the breached company's first letter)
+ */
+export function getBreachLogo (breach, logos) {
+  const logo = logos.has(breach.Domain)
+    ? `<img src='${logos.get(breach.Domain)}' alt='' class='breach-logo' height='32' />`
```

Comment:
Cool that this can be repurposed for both instances!  

For the all-breaches instance, the Monitor logo in the header (and other loads like fonts) are noticeably delayed for me because of all the image requests.  I think a `loading="lazy"` attribute in the image tag would resolve that.